sirepo/srunit.py

```diff
@@ -6,11 +6,14 @@
 """
 from __future__ import absolute_import, division, print_function
 from pykern import pkcompat
+from pykern import pkjson
```

Comment:
Move this down where it is needed.

Reply:
https://github.com/radiasoft/sirepo/commit/66da4701448010669a6cbfea717df20863acec62